tests/sqllogictests/suites/mode/cluster/exchange.test

```diff
@@ -177,3 +177,74 @@ Exchange
             ├── partitions scanned: 1
             ├── push downs: [filters: [], limit: NONE]
             └── estimated rows: 2.00
+
+query T
+explain fragments select * from (select sum(number) as number from numbers(1) group by number) t, numbers(2) t1 where t.number = t1.number
+----
+Fragment 0:
+  DataExchange: Shuffle
+    ExchangeSink
+    ├── destination fragment: [3]
+    └── TableScan
+        ├── table: default.system.numbers
+        ├── read rows: 2
+        ├── read bytes: 16
+        ├── partitions total: 1
+        ├── partitions scanned: 1
+        ├── push downs: [filters: [], limit: NONE]
+        └── estimated rows: 2.00
+(empty)
```

Comment:
What's `(empty)` mean? Can we ignore it when print.

Reply:
It represents an empty string, the line breaks. This can help distinguish between fragments.